apps/src/templates/sectionProgressV2/section-progress-refresh.scss

```diff
@@ -0,0 +1,65 @@
+.legend {
+    align-items: flex-start;
+    border: 1px solid;
+    border-color: var(--lightgray-200);
+    border-radius: 4px;
+    display: inline-flex;
+    flex-direction: column;
+    flex: 1 1 auto;
+  }
+
+  .legend .headerContainer {
```

Comment:
It looks like this might be a different style of css definitions than I'm used to. For coteacher, I was using this style that would `import styles from 'file.module.scss`. Is there a reason we should be using this?

Reply:
File name should probably be `section-progress-v2.module.scss`. You can also use [this](https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/templates/sectionProgressV2/progress-table-v2.module.scss) file or create a new one that is specific to legend

Reply:
Converted to `.module.scss` format.